### PR TITLE
🖕😏🖕

### PR DESCRIPTION
**Pull request by rowe-morehouse**
🖕😏🖕

### CODE REVIEW

**Comment by Shuunen** [comment]
this repo is cleary un-maintained

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # I rudely reject pull requests to this repo
 Its fucking perfect as it is.
+<span title="🖕😏🖕"></span>
```
Comment (Shuunen) [inline README.md:3]: I would rather go for `<span title="🖕">😏</span>`

**Review by Shuunen — CHANGES_REQUESTED** [review]
something more classy please

**Comment by Shuunen** [comment]
@jakearchibald must be compressing bitmap or svg again, don't blame him I have the same OCD

**Comment by jakearchibald** [comment]
this didn't even deserve to be rejected in a timely fashion

**Comment by Shuunen** [comment]
Nooooooo

**Pull request closed without merging**